up42/utils.py

```diff
@@ -338,7 +343,7 @@ def filter_jobs_on_mode(jobs_json: List[dict], test_jobs: bool = True, real_jobs
     if not selected_modes:
         raise ValueError("At least one of test_jobs and real_jobs must be True.")
     jobs_json = [job for job in jobs_json if job["mode"] in selected_modes]
-    logger.info(f"Returning {selected_modes} jobs.")
+    logger.info("Returning %s jobs.", selected_modes)
```

Comment:
This can be moved to L345 and the jobs_json can be inlined

Reply:
I didn't get `jobs_json can be inlined` fully.
Please see if the changes represent what you expected.